ota-plus-web/app/reactapp/src/components/device/HardwarePanel.jsx

```diff
@@ -80,7 +80,7 @@ class HardwarePanel extends Component {
                          onClick={this.showSecondaryDescription} id="hardware-secondary-ecu-details" 
                      />
                 </div>
-                {!_.isEmpty(device.directorAttributes.secondary) ?
+                {!_.isEmpty(device.directorAttributes.secondary) || device.directorAttributes.secondary.length ?
```

Comment:
How can the second check pass without the first?

Reply:
First check does not pass when we have observable array(mobx thing). I should think how to refactor this a bit later